src/display/macros.rs

```diff
@@ -0,0 +1,176 @@
+/// |                        | ├─────────┼──────────┤ |
+/// |                        | │ Manjaro │ Arch     │ |
+/// |                        | └─────────┴──────────┘ |
+/// +------------------------+------------------------+
+/// */
+/// ```
+///
+/// ### Table Duplication
+///
+/// Akin to the `vec![T; N]` pattern for repetition.
+///
+/// ```rust,no_run
+/// println!("{}", group!(table_a; 3));
+/// /*
+/// +------------------------+------------------------+------------------------+
+/// | .--------------------. | .--------------------. | .--------------------. |
+/// | | name    | based_on | | | name    | based_on | | | name    | based_on | |
+/// | | Debian  |          | | | Debian  |          | | | Debian  |          | |
+/// | | Arch    |          | | | Arch    |          | | | Arch    |          | |
+/// | | Manjaro | Arch     | | | Manjaro | Arch     | | | Manjaro | Arch     | |
+/// | '--------------------' | '--------------------' | '--------------------' |
+/// +------------------------+------------------------+------------------------+
+/// */
+/// ```
+///
+/// ### Rows
+///
+/// - Each rows height is calculated separately.
+/// - Each columns width is calculated separately.
+/// - Rows are chunked by the defined increment.
+/// - Empty cells default to blank.
+///
+///  ```rust,no_run
+/// println!("{}", group!(table_a, table_b, table_c; 2));
+/// /*
+/// +-------------------------------------+------------------------+
+/// | .--------------------.              | ┌─────────┬──────────┐ |
+/// | | name    | based_on |              | │ name    │ based_on │ |
+/// | | Debian  |          |              | ├─────────┼──────────┤ |
+/// | | Arch    |          |              | │ Debian  │          │ |
+/// | | Manjaro | Arch     |              | ├─────────┼──────────┤ |
+/// | '--------------------'              | │ Arch    │          │ |
+/// |                                     | ├─────────┼──────────┤ |
+/// |                                     | │ Manjaro │ Arch     │ |
+/// |                                     | └─────────┴──────────┘ |
+/// +-------------------------------------+------------------------+
+/// | | name                 | based_on | | ┌─────────┬──────────┐ |
+/// | |----------------------|----------| | │ name    │ based_on │ |
+/// | | Super Long Name Here |          | | ├─────────┼──────────┤ |
+/// | | Arch                 |          | | │ Debian  │          │ |
+/// | | Manjaro              | Arch     | | ├─────────┼──────────┤ |
+/// |                                     | │ Arch    │          │ |
+/// |                                     | ├─────────┼──────────┤ |
+/// |                                     | │ Manjaro │ Arch     │ |
+/// |                                     | └─────────┴──────────┘ |
+/// +-------------------------------------+------------------------+
+/// */
+/// ```
+///
+/// ### Clean Display
+///
+/// The `@` identifier is recognized by all patterns to produce
```

Comment:
Personally if I'd use the library I'd take `@` to be something a bit strange :)

I think it's not nessary?
I mean if someone would like to make it `clean`  he/she could just use `with.(Style::empty())`.
Am I wrong here?

Reply:
I completely agree.

I will *clean* this up 👍🏻